cmd/frontend/graphqlbackend/external_services_test.go

```
@@ -236,6 +237,7 @@ func TestAddExternalService(t *testing.T) {
 					"kind": "GITHUB",
 					"displayName": "GITHUB #1",
 					"config": "{\"url\": \"https://github.com\", \"repositoryQuery\": [\"none\"], \"token\": \"abc\"}",
+					"config":"{\n  \"url\": \"https://github.com\",\n  \"repositoryQuery\": [\n    \"none\"\n  ],\n  \"token\": \"` + types.RedactedSecret + `\"\n}",
```

A bit confused that we have two occurrences of `"config"`?

---

oh weird, i guess the json parser silently swallows the first instance. i'll delete